src/stores/AnnotationStore.js

```diff
@@ -840,7 +840,7 @@ const Annotation = types
         });
       }
 
-      self.objects.forEach(obj => obj.needsUpdate?.());
+      self.updateObjects();
```

Comment:
`updateObjects` contains `unselectAll`, so I'm afraid that this change may break the user experience or even the functional of suggestings in case of working with the ml-backend.